Add support for multiple keys

flosell opened the pull request:
At the moment, when having more than one `hcl:",key"`, one key overwrites the other. I would expect multiple keys to be possible to generate things that look like terraform resources. 

This PR fixes this issue.

flosell committed:
Add support for multiple keys

rodaine commented:
Hey, @flosell! Thanks for the PR. Looks like your change to the vet command in the travis config is causing CI to fail: https://travis-ci.org/rodaine/hclencoder/builds/372362857#L520

flosell commented:
Sorry, that was never supposed to be part of the PR! I updated the commit but it looks like travis built the old one again (in retrospect, maybe force-pushing the PR branch wasn't such a good idea :) didn't want to have this noise in the git repo)

avorima commented:
Hey @flosell, any chance you could look into resolving the issue with script/test? I'd really like to use your changes.\
Thanks.

promiseofcake commented:
These updates to script/test should make it forward/backward compatible-ish as Go versions move. `-shadow` is a part of `-all` in recent versions.

```diff
diff --git a/script/test b/script/test
index 8ea5943..6242a1f 100755
--- a/script/test
+++ b/script/test
@@ -4,11 +4,16 @@ set -e
 echo "go fmt..."
 test -z "$(gofmt -l -w . | tee /dev/stderr)"
 
+if ! [ -x "$(command -v golint)" ]; then
+  echo "installing golint..."
+  go get -u golang.org/x/lint/golint
+fi
+
 echo "go lint..."
 test -z "$(golint ./... | tee /dev/stderr)"
 
 echo "go vet..."
-test -z "$(go tool vet -test . 2>&1 | tee /dev/stderr)"
+test -z "$(go tool vet -all -shadow . 2>&1 | tee /dev/stderr)"
 
 echo "go test..."
 go test -race -cover ./...
```

flosell committed:
Rename variable to avoid shadowing previous variable

flosell commented:
@avorima and @promiseofcake thanks for the reminder! 

I rebased the changes on the current master and fixed the `go vet` finding, this fixed the build. I submitted a separate PR for the fixes in `script/test` to keep the scopes separate (lint is green anyway for this PR)

rodaine commented:
Awesome, thanks for the patch!

avorima commented:
Thanks!
